tests/Php82/Php82Test.php

```diff
@@ -84,4 +91,363 @@ public static function provideMoreConnectionStringValues(): \Generator
     {
         yield 'double curly at the end' => ['foo}}', false, true, '{foo}}}}}'];
     }
+
+    /**
+     * @covers \Symfony\Polyfill\Php82\Php82::ini_parse_quantity
+     */
+    public function testIniParseQuantity()
+    {
+        $this->assertSame(0, Php82::ini_parse_quantity(''));
```

Comment:
Tests should use `ini_parse_quantity` (but relying on namespace fallback, not throught a fully qualified class name), not the internal implementation method. This allows the TestListener to execute the test twice, using the native implementation or the polyfill.

Reply:
Done